app/src/main/java/com/wtmberlin/util/ErrorLogger.kt

```diff
@@ -0,0 +1,10 @@
+package com.wtmberlin.util
+
+class ErrorLogger {
+
+    fun getException(e: Throwable) {
+        //Todo: add a crash lib
+        val a = e
```

Comment:
What do you think @ashdavies

Reply:
Conditional reporting can be achieved with a debug `Timber.Tree` which can always be beneficial, but crash reporting should be configured by the project owner since it requires setup of an additional service and an API key which can sometimes incur billing.